build.sbt

```diff
@@ -9,6 +9,8 @@ organization := "org.broadinstitute"
 
 scalaVersion := "2.11.7"
 
+val lenthallV = "0.13-eb2dd2d-SNAPSHOT"
```

Comment:
We're using our own semi-SemVer and manually updating build numbers. Builds are always snapshots until we tag a release at `0.13`, `0.14`, etc. So 0.13-**eb2dd2d**-SNAPSHOT is the current HEAD version of https://github.com/broadinstitute/lenthall/tree/develop